src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -165,6 +165,10 @@ public function getFlashes($types = null)
             return [];
         }
 
+        if (!method_exists($session, 'getFlashBag')) {
```

Comment:
```suggestion
        if (null === $session || !method_exists($session, 'getFlashBag')) {
```

In order to keep the code compact, I'd merge your check with the one inside the try/catch statement. And since that null check won't trigger an exception, it should be save to pull it out of the try block.

Reply:
the return type for `getSession` should be `SessionInterface|null` (the null case is deprecated btw, so this needs cleanup in 5.0 as well)

Reply:
Fixed, and the return type for `getSession` as well, @ro0NL :slightly_smiling_face: